### DISCUSSION TITLE
[Feature Request] Contacts List per Alias like SimpleLogin

### DISCUSSION THREAD

**Discussion by ghost** [anonaddy/anonaddy#402] [Feature Requests]
Hello everyone,

so I came over to AnonAddy from SimpleLogin today, the only thing I am missing and would kindly request to be added in the future to AnonAddy is the contacts feature of an alias from SimpleLogin. Like a list (per alias) of all senders that have send a mail to that alias, containing an automatically created reverse alias for each sender to reply to that sender without needing to create it yourself AND, most importantly, to block specific senders that send a mail once to that alias, to keep your alias free from spam and unwanted senders/mails (e.g. if your alias got compromised because the service it is registered with got breached but you don't want to change the alias and want to keep using the service/site with the corresponding alias). I really like that feature from Simple and think it's very useful to be also added to Anon.

Thanks in advance for taking your time to read my suggestion (and maybe replying/considering it)

Cheers!

**Comment by willbrowningme** [comment]
Thanks for the suggestion! My main issue with this is that it means logging and storing details about every single message sent through the service indefinitely (since someone may wish to reply to an alias a couple of years later).

You can simply click reply to any forwarded message in AnonAddy, you do not need to create the correct email to reply to yourself. That is only needed when you initiate a conversation, and then you can use the send from "address constructor" on the web app, extension and apps.

I agree the option to block specific sender per alias would be a great addition and is something I do plan to add in the future.

**Reply by ghost** [reply]
Hey!

Appreciate that you took your time to read my suggestion and answer to it. Glad to hear that blocking is on your To-do List.

Cheers and have a nice day!

**Comment by vertigo220** [comment]
@Paddieh I'm currently looking at this service and SimpleLogin and am curious why you've chosen to move over from there to here.

**Reply by Arpi** [reply]
I have tested the SimpleLogin service thoroughly and can tell you that (besides some successful deliveries) there were some failed deliveries there, despite setting up everything as per their suggestions and guides. 
Furthermore, I have sent them detailed mail server logs (taken from the mail server at my employer) indicating that those non-delivered, i.e. lost messages were successfully sent from our mail server. The only answer for that was a meaningless reply from a dumb support staff member asking for more info, despite the fact that the maximum and most detailed info was already provided in the form of the server message log. 

Obviously, failed message deliveries is an absolute red flag and no-no in any email service, so one should choose any service, but NOT SimpleLogin. 

I am very satisfied with AnonAddy and have no complaints.
